MatrixSDK/VoIP/MXCall.m

```diff
@@ -338,31 +256,40 @@ - (void)callWithVideo:(BOOL)video
             NSLog(@"[MXCall] callWithVideo:%@ - Offer created: %@", (video ? @"YES" : @"NO"), sdp);
 
             // The call invite can sent to the HS
-            NSDictionary *content = @{
-                                      @"call_id": self.callId,
-                                      @"offer": @{
-                                              @"type": @"offer",
-                                              @"sdp": sdp
-                                              },
-                                      @"version": @(0),
-                                      @"lifetime": @(self->callManager.inviteLifetime)
-                                      };
+            NSMutableDictionary *content = [@{
+                @"call_id": self.callId,
+                @"offer": @{
+                        @"type": kMXCallSessionDescriptionTypeOffer,
+                        @"sdp": sdp
+                },
+                @"version": kMXCallVersion,
```

Comment:
It can be both. Spec says: 

> clients must accept either a number or a string for the value of the version field, in order to allow for namespaced versions in the future.